Dfe.Academies.Academisation.Data/Repositories/ApplicationRepository.cs

```diff
@@ -78,5 +78,16 @@ private IQueryable<Application> DefaultIncludes()
 
 			return x;
 		}
+
+		public async Task SoftDelete(int applicationId)
+		{
+			var entity = await _context.Applications.FindAsync(applicationId);
```

Comment:
This function could use a bit more error handling. Right now, if applicationId does not exist in the database, the function silently does nothing. You might want to add some way to notify the caller that the delete operation failed.

Reply:
we have some error handling a of not found at the command layer and unit tests to verify it... NotFound___NotPassedToDataLayer_NotFoundReturned()